Project.toml

```diff
@@ -13,9 +13,12 @@ GaussianProcesses = "891a1506-143c-57d2-908e-e1f8e92e6de9"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Optim = "429524aa-4258-5aef-a3af-852621145aeb"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
Likely, CES.jl shouldn't directly depend on Plots (#65). Rather, we can make a test environment (https://julialang.github.io/Pkg.jl/dev/creating-packages/#Test-specific-dependencies-in-Julia-1.2-and-above-1).

Reply:
Yup this is what we are thinking - We are planning on merging this PR first, then doing a 'CES' cleanup to deal with the `project.toml` 's and alike to put dependencies only where required

Reply:
@charleskawczynski, that's right, but we will address #65 with a separate PR following this one. In that "clean-up PR", we will create separate `Project.toml` and `Manifest.toml` files for each example, with the idea that this will get rid of some annoying dependencies that are only needed for running the examples, but not for the core CES functionality. There are also quite a few modules in `src/` that are not functional and are not being developed any further -- removing them will be part of the cleaning as well.

Reply:
My comment came too late :-)